go.mod

```diff
@@ -3,7 +3,7 @@ module go.uber.org/automaxprocs
 go 1.13
 
 require (
+	github.com/kr/pretty v0.1.0 // indirect
```

Comment:
You should be able to remove these indirect dependencies as I believe they're only necessary in the submodule now
https://github.com/uber-go/automaxprocs/pull/32/files#diff-3295df7234525439d778f1b282d146a4f1ff6b415248aaac074e8042d9f42d63L7

Reply:
We can't remove this; it's an indirect dependency of testify:

```
$ go mod why github.com/kr/pretty
# github.com/kr/pretty
go.uber.org/automaxprocs/internal/cgroups
go.uber.org/automaxprocs/internal/cgroups.test
github.com/stretchr/testify/assert
gopkg.in/yaml.v2
gopkg.in/yaml.v2.test
gopkg.in/check.v1
github.com/kr/pretty
```